packages/office-addin-dev-settings/src/sideload.ts

```diff
@@ -82,16 +90,7 @@ export async function generateSideloadFile(app: OfficeApp, manifest: ManifestInf
     .replace(/1.0.0.0/g, manifest.version);
   zip.file(webExtensionPath, webExtensionXml);
 
-  // Write the file
-  await new Promise((resolve, reject) => {
-    zip
-      .generateNodeStream({ type: "nodebuffer", streamFiles: true })
-      .pipe(fs.createWriteStream(pathToWrite))
-      .on("error", reject)
-      .on("finish", resolve);
-  });
-
-  return pathToWrite;
+  return await writeZipFile(zip, pathToWrite);
```

Comment:
Does this still open the host application with the provided file?

Reply:
Yes, the provided file is opened